### ISSUE TITLE
Can't import resources in cura 2.7

### ISSUE THREAD

**Issue by dr4Ke** [FABtotum/FAB_Configs#12]
With a fresh installation of Cura 2.7, and after having copied Fabtotum resources in it, I have the following warnings:

- WARNING - cura.Settings.CuraContainerRegistry.addContainer [54]: Instance container fabtotum_pla_wood_normal is outdated. Its setting version is 1 but it should be 2.
- WARNING - cura.Settings.CuraContainerRegistry.addContainer [54]: Instance container fabtotum_pla_blue_fast is outdated. Its setting version is 1 but it should be 2.
- etc.

It seems related to this Cura issue: https://github.com/Ultimaker/Cura/issues/2117
I guess we have to update the Fabtotum resources definitions to match this new cura version.

**Comment by dr4Ke** [comment]
Just saw pull request #11 for this one.

**Comment by ClemaX** [comment]
I don't think this is closely related. #2117 seems to be caused by missing translation for legacy profiles (translation does not provide setting version so it defaults to 0).
